dateconv/dateconv.go

```diff
@@ -134,6 +134,32 @@ func TotalDaysInBSYear(year int) (int, error) {
 	return sum, nil
 }
 
+// now returns the current time
+var now = func() time.Time {
+	return time.Now()
+}
+
+// TotalDaysSpanned returns the total number of days spanned in the
+// current year inclusive of the current day.
+func TotalDaysSpanned() (int, error) {
```

Comment:
Slightly worried that the package name is starting to not reflect the functionality that it actually provides.

Reply:
Yes, I was thinking the same. We need to move some of this utility to a new package like `util` or something. `dateconv` should only have conversion utilities.

Reply:
I can do this in a new PR if you feel we should do it.

Reply:
Leave it to me I guess. I don't like `util` packages in Go 😏

Reply:
Woahhh ... It is actually "nepcal/utilities".

Reply:
I understand but the nesting doesn't matter to the package user because it's still going to be `utilities.TotalDaysSpanned()` without any `nepcal` namespace?

Reply:
Moving to #33